tests/environment/test-setup/Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM bitnami/openldap:latest
+FROM bitnami/openldap:2.5.18
 
 USER root
 
+RUN echo 'deb http://deb.debian.org/debian bookworm-backports main' >> /etc/apt/sources.list
+
 RUN apt-get update && apt-get upgrade -y && \
-	apt-get install --yes curl golang-go jq && \
+	apt-get install --yes curl jq && \
+	apt-get install --yes -t bookworm-backports golang-go && \
```

Comment:
Why do we need older go compiler?

Reply:
Newer, backports come from unstable. The zitadel tools package depends on go 1.21 now, but debian bookwork packages 1.20.